### ISSUE TITLE
Model pre-trained on the National Corpus of Polish and Morfeusz version

### ISSUE THREAD

**Issue by djstrong** [kawu/concraft-pl#45]
National Corpus of Polish is not compatible with "current version of Morfeusz SGJP (i.e., the version from September 1st 2018 or newer)", e.g. in NCP is tag `qub`, but in Morfeusz is tag `part`. So how it was trained?

**Comment by kawu** [comment]
To train the model, colleagues from ICS PAS (Kasia Krasnowska-Kieraś, Witek Kieraś) developed a script to automatically merge the result of morphosyntactic analysis (Morfeusz output) with NCP.  I'm not sure if this script was released, though.

**Comment by djstrong** [comment]
The script should translate "old" tagset/annotation (NCP) to "new" (current Morfeusz).
I tried to evaluate shared model on PolEval 2017 data, but there is the problem with incompatible tagset - PolEval2017 data is annotated with older guidelines.

**Comment by kawu** [comment]
Yes, indeed, the script developed at ICS PAS translated NCP annotations to Morfeusz-compliant annotations.  But this was not a simple old-to-new tagset conversion, it was more like projecting NCP annotations on Morfeusz graphs.

Concerning PolEval 2017, since the tagset is not compatible with Morfeusz, evaluating Concraft on it may be tricky.  Of course you could also train Concraft directly on NCP, but that would require converting NCP to the format required by Concraft.

Also note that Concraft is not based on neural methods, I doubt it can give you competitive results with neural systems from the PolEval 2017 (tagging) shared task.  The current version of Concraft is specifically designed to be used with Morfeusz, which may be more difficult with other existing taggers for Polish.  I am actually planning to rewrite it in PyTorch, but that will require some time I don't have at the moment...
